ctw/japan/map.xml

```diff
@@ -35,62 +33,46 @@
 <spawns>
     <default>
         <regions>
-            <point yaw="0">18.5,22.02,-49.5</point>
+            <point>18.5,22.5,-49.5</point>
         </regions>
     </default>
-    <spawn team="green" kit="spawn-kit">
+    <spawn team="green" kit="spawn-kit" yaw="-90">
```

Comment:
You should change here `<regions/>` in this to `<region/>`; `<regions/>` is for collections of point regions that can have individual yaws.

I would prefer that you keep the yaw within the `<region/>` as defined here, because yaws can be per spawn region.

So, for this:


```xml
    <spawn team="green" kit="spawn-kit">
        <region yaw="-90">
            <point>62.5,11,21.5</point>
        </region>
    </spawn>
```